### PR TITLE
OBPIH-1068: Fix saving stocklist and refactor the name of this column

### PR DESCRIPTION
**Pull request by pmuchowski**

### CODE REVIEW

File: src/js/components/stock-movement-wizard/CreateStockMovement.jsx [hunk 1]
```diff
@@ -181,17 +182,19 @@ class CreateStockMovement extends Component {
...
     const isNewSupplier = newValues.origin && newValues.type === 'SUPPLIER';
     const checkOrigin = originLocs && (!isOldSupplier || (isOldSupplier && !isNewSupplier)) ?
       newValues.origin.id !== origin.id : false;
 
-    const checkDest = stockList && newValues.destination && destination ?
+    const checkDest = stocklist && newValues.destination && destination ?
       newValues.destination.id !== destination.id : false;
-    const checkStockList = newValues.stockMovementId ? newValues.stockList !== stockList : false;
+    const checkStockList = newValues.stockMovementId ? _.get(newValues.stocklist, 'id') !== _.get(stocklist, 'id') : false;
+    console.log('new stocklist id: ', _.get(newValues.stocklist, 'id'));
```
Comment (awalkowiak) [inline src/js/components/stock-movement-wizard/CreateStockMovement.jsx]: console.log

File: src/js/components/stock-movement-wizard/CreateStockMovement.jsx [hunk 2]
```diff
@@ -181,17 +182,19 @@ class CreateStockMovement extends Component {
...
+    console.log('old stocklist id: ', _.get(stocklist, 'id'));
```
Comment (awalkowiak) [inline src/js/components/stock-movement-wizard/CreateStockMovement.jsx]: console.log

File: src/js/components/stock-movement-wizard/CreateStockMovement.jsx [hunk 3]
```diff
@@ -223,6 +226,7 @@ class CreateStockMovement extends Component {
    * @public
    */
   saveStockMovement(values) {
+    console.log('save vals: ', values);
```
Comment (awalkowiak) [inline src/js/components/stock-movement-wizard/CreateStockMovement.jsx]: console.log

File: src/js/components/stock-movement-wizard/CreateStockMovement.jsx (hunk 1 above)
Comment (pmuchowski) [reply src/js/components/stock-movement-wizard/CreateStockMovement.jsx]: I will remove this

**Pull request merged**